agent/eni/watcher/watcher_linux.go

```diff
@@ -184,6 +189,11 @@ func (udevWatcher *UdevWatcher) reconcileOnce() error {
 	// Add new interfaces next
 	for mac := range currentState {
 		if err := udevWatcher.sendENIStateChange(mac); err != nil {
+			// skip logging status sent error as it's redundant and doesn't really indicate a problem
+			if strings.Contains(err.Error(), eniStatusSentMsg) {
+				continue
+			}
```

Comment:
Will this show the log message with ENI trunking disabled?

Reply:
no. but i don't think it needs to show in that case either?

Reply:
Sounds good.